cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -154,6 +154,16 @@ Known Issues
     
   All versions of CDAP running Twill version 0.4.0 with this configuration can exhibit this
   problem (`TWILL-110 <https://issues.apache.org/jira/browse/TWILL-110>`__).
+- Typically Datasets are bundled as part of Applications. When an Application is upgraded and redeployed,
+  any changes in Datasets will not be redeployed. This is because Datasets can be shared across applications,
+  and an incompatible schema change can break other applications that are using the Dataset.
+  A workaround (`CDAP-1253 <https://issues.cask.co/browse/CDAP-1253>`__) is to set a configuration,
```

Comment:
A workaround (`CDAP-1253 <https://issues.cask.co/browse/CDAP-1253>`__) is to set configuration `dataset.unchecked.upgrade` to `true` in `cdap-site.xml`. This will ensure that Datasets will always get redeployed when an Application is redeployed.